### DISCUSSION TITLE
"tools" vs "toolchain" is confusing

### DISCUSSION THREAD

**Discussion by sfermigier** [astral-sh/rye#294] [Ideas]
I think it's confusing to have both "tools" and "toolchain" concepts (specially for non-native English speakers).

Same for the duality "add" / "install" and "remove" / "uninstall". 

(As reference points, we have "apt install / remove", "brew install / uninstall", "pkg add / delete" (on FreeBSD), "apk add / del", etc.)

I'd suggest to rename "tools" -> "globals" and to remove "install" and "uninstall" from the top-level commands (as, IIUC, they are also provided as subcommands the "tools" top-level command).

**Comment by mitsuhiko** [comment]
I'm not opposed to renaming `tools` but I'm not sold on the replacement `globals`. Maybe something better can be found here that is more descriptive.

**Comment by cemrehancavdar** [comment]
Similar to "pipx," we can refer to them as applications or apps. 
And / or we can use ryx as in npx for npm. It might be confusing for new comers but will distinguish it.
